package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "t2-compiler",
+  "version": "1.0.0",
+  "description": "Server that compiles modules for the tessel platform",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tessel/t2-compiler.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/tessel/t2-compiler/issues"
+  },
+  "homepage": "https://github.com/tessel/t2-compiler#readme",
+  "dependencies": {
+    "bluebird": "^3.1.1",
+    "concurrent-couch-follower": "^1.0.1",
+    "got": "^6.0.1",
+    "gunzip-maybe": "^1.2.1",
+    "tar": "^2.2.1"
+  }
+}
```

Comment:
I'm going to use this exact file as the starting point for the tooling additions. **Hopefully** when you rebase, this will become a clean patch/diff